lib/clients/BaseClient.py

```diff
@@ -70,6 +71,28 @@ def __init__(self, operation_name, configuration, directory_persistent, director
         self.__mounted_devices = []
         self.__devices = {}
 
+    def _get_credentials_from_credhub(self, configuration):
+        access_token = self.__getAccessToken(configuration)
+        params = {'name': configuration['credhub_key']}
```

Comment:
Shall we also say current as yes here so that we get the latest credentials from credhub?